en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,812 @@
+    
+    2. Enter information for the variable as follows:
+    
+        | **Field**         | **Value**        |
+        |-------------------|------------------|
+        | **Type**          | **other**        |     
+        | **Other Type**    | `string[]`       |
+        | **Name**          | `respondMessages`|
+        | **Expression**    | `[]`             |
+        
+    3. Click **Save**.
+        
+ 10. To log the response, add a `log` statement as follows:
+ 
+    1. Click the **+** below the `respondMessages` variable statement you added, and then click **Log**.
+    
+    2. In the **Expression** field, enter `"itemIDs.toJsonString()`. 
+    
+    3. Click **Save**.    
+    
+11. To define a parameter for the inventory record, add a new variable as follows:
+
+    1. Click the **+** below the log statement you added, and then click **Variable**.
+    
+    2. Enter information for the variable as follows:
+    
+        | **Field**      | **Value** |
+        |----------------|-----------|
+        | **Type**       | **int**   |     
+        | **Name**       | `i`       |
+        | **Expression** | `1`       |
+        
+    3. Click **Save**.
+        
+12. To define a parameter for the inventory index, add a new variable as follows:
+    
+    1. Click the **+** icon below the last `i` variable statement you added, and then click **Variable**.
+    
+    2. Enter information for the variable as follows:
+    
+        | **Field**      | **Value** |
+        |----------------|-----------|
+        | **Type**       | **int**   |     
+        | **Name**       | `index`   |
+        | **Expression** | `-1`      |
+        
+    3. Click **Save**.
+        
+13. To define a parameter for the item ID, add a string variable as follows:   
+
+    1. Click the **+** icon below the last `index` variable statement you added, and then click **Variable**.
+    
+    2. Enter information for the variable as follows:  
+ 
+         | **Field**      | **Value**      |
+         |----------------|----------------|
+         | **Type**       | **string**     |     
+         | **Name**       | `itemIDString` |
+         | **Expression** | `""`           | 
+         
+    3. Click **Save**.
+         
+14. Add a `foreach` statement as follows:
+   
+    1. Click the **+** below the last `itemIDString` variable statement you added, and then click **ForEach**.
+    
+    2. Enter information for the ForEach statement as follows:
+    
+         | **Field**                  | **Value**   |
+         |----------------------------|-------------|
+         | **Current Value Variable** | **ItemID**  |     
+         | **Iterable Expression**    | `itemIDs`   |
+         
+    3.Click **Save**.
+         
+    4. To specify the action to be iterated for each order request, add a custom statement within the ForEach statement as follows:
+     
+        1. Click the **+** icon below the `foreach` statement you added.
```

Comment:
Here we should click on the `+` icon inside the `foreach` statement isn't it?

Reply:
By "below", I meant the + icon just below the square. There is a supporting screenshot with the relevant + icon marked on it.